### DISCUSSION TITLE
在wayland compositor里面,text input激活时,可以不用按shift键,自动显示"en"或"拼"的提示吗?

### DISCUSSION THREAD

**Discussion by guyuming76** [fcitx/fcitx5#1068] [Q&A] [answered]
我在知乎里答了个题, 有些地方没把握, 还有些细节没搞清: [https://www.zhihu.com/question/19695419/answer/3512527140](https://www.zhihu.com/question/19695419/answer/3512527140) ,能否帮忙看看?

https://wayland.freedesktop.org/extras.html 列了一系列工具,哪个可以用来帮助解读wayland compositor 和fcitx5 之间的通讯吗?

**Comment by wengxt** [comment, marked as answer]
全局选项里设置在焦点更改时显示输入法信息

看消息用环境变量 WAYLAND_DEBUG=1 即可
